### PR TITLE
Event Calendar List

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/organisms/CalendarList.tsx [hunk 1]
```diff
@@ -58,19 +58,27 @@ const orderList = css({
 interface CalendarListProps {
   calendars: ReadonlyArray<CalendarResponse>;
   singleGroup?: boolean;
```
Comment (jeysal) [inline packages/react-components/src/organisms/CalendarList.tsx:60]: Looks like these are exclusive, specifying both doesn't make sense, so the interface would need to change. But instead maybe just pass in the headline and a bool for whether to show the explanation text?

File: packages/react-components/src/organisms/CalendarList.tsx [hunk 2]
```diff
@@ -58,19 +58,27 @@ const orderList = css({
...
   singleGroup = false,
+  singleEvent = false,
 }) => (
   <div css={containerStyles}>
     <Card>
       <div css={headerStyles}>
         <Headline3>
-          Subscribe to {singleGroup ? "this group's" : 'Groups on'} calendar
+          Subscribe to{' '}
+          {singleGroup
```
Comment (migcarva) [inline packages/react-components/src/organisms/CalendarList.tsx:73]: Can we just change the order of this? check `singleEvent`, then `singleGoup`

File: packages/react-components/src/organisms/CalendarList.tsx [hunk 3]
```diff
@@ -57,20 +57,24 @@ const orderList = css({
 });
 interface CalendarListProps {
   calendars: ReadonlyArray<CalendarResponse>;
-  singleGroup?: boolean;
+  page: 'calendar' | 'event' | 'group';
+  singleEvent?: boolean;
```
Comment (jeysal) [inline packages/react-components/src/organisms/CalendarList.tsx:61]:
```suggestion
```

**Review by jeysal — APPROVED** [review]
one cleanup then lgtm

**Pull request merged**